models/convert/cadence.go

```diff
@@ -182,9 +181,13 @@ func ParseCadenceArgument(param string) (cadence.Value, error) {
 }
 
 // ParseRosettaValue parses the input string, which should be the Rosetta value, and
-// produces the Cadence UFix64 number. The function will assume that the input string
+// produces the Cadence UFix64 number. The function verifies that the input string
 // has dps.FlowDecimals decimal places.
-func ParseRosettaValue(value string) (cadence.UFix64, error) {
+func ParseRosettaValue(value string, fractionLen uint) (cadence.UFix64, error) {
```

Comment:
I misunderstood how these values are represented in Rosetta (I thought they were coming with a decimal). Given that they are defined as the smallest atomic unit, this function is completely unnecessary.

We simply need to parse the value as `uint64` and then cast it to `cadence.UFix64`, see here:
https://github.com/onflow/cadence/blob/fa0c3f625056cbaf001acc8567658c8aaabdbffb/values.go#L885